src/postprocessing/idlparsed.js

```diff
@@ -8,17 +8,170 @@
 const webidlParser = require('../cli/parse-webidl');
 
 module.exports = {
-  dependsOn: ['idl'],
+  dependsOn: ['dfns', 'idl'],
   input: 'spec',
   property: 'idlparsed',
 
   run: async function(spec, options) {
+    function getHref(idl, member) {
+      let dfnType;
+      let dfnFor;
+      let dfnOverload = 0;
+      let dfnName;
+      if (member) {
+        if (['iterable', 'maplike', 'setlike'].includes(member.type) ||
+            ['getter', 'setter', 'stringifier', 'deleter'].includes(member.special)) {
+          // No dfns of these types in any spec as of Feb 2024, or at least no
+          // no dfns that we can easily map to (for example, the HTML spec
+          // tends to use generic "dfn" for these).
+          return null;
+        }
+        if (member.type === 'operation') {
+          dfnType = 'method';
+          dfnOverload = idl.members
+            .filter(m => m.type === member.type && m.name === member.name)
+            .findIndex(m => m === member);
+        }
+        else if (member.type === 'field') {
+          dfnType = 'dict-member';
+        }
+        else {
+          dfnType = member.type;
+        }
+        if (!['constructor', 'method', 'attribute', 'enum-value', 'dict-member', 'const'].includes(dfnType)) {
+          console.error(`[error] Found unexpected IDL member type "${dfnType}" in ${spec.shortname}`);
+        }
+        dfnName = member.name ?? member.value;
+        dfnFor = idl.name;
+      }
+      else {
+        // The type of the dfn to look for is the same as the IDL type, except
+        // that composed IDL types ("interface mixin", "callback interface")
+        // only have the basic type in definitions.
+        dfnType = idl.type.split(' ')[0];
```

Comment:
since you have a console.error for unknown member type above, shouldn't there be one for unknown idl type here? esp. since this relies on the fairly brittle pattern of space-splitting the type…

(there is probably a more general question of making reffy more resistant to IDL evolutions, but I'm not quite sure how to approach it atm)